### ISSUE TITLE
Output TS files have system default encoding

### ISSUE THREAD

**Issue by ADegele** [cincheo/jsweet#413]
Output Files ignore the encoding configuration for the transpiler. I am using a windows machine, so the platform encoding is ANSI. In the configuration of jsweet I've included the `<encoding>UTF-8</encoding>`, which is working for input files as far as I can tell, otherwise the transpiler would mangle our UTF-8 encoded Java Files.
However, TS Output Files are encoded with UTF-8. Umlaute in strings are properly escaped, but comments are not. Following is an excerpt from one of our files: 
```
/**
* Bankfreie Betreuung._t Für Werbung ansprechbar
*/
public static NR_0 : AnsprechSchluesselVertragEnum; public static NR_0_$LI$() : AnsprechSchluesselVertragEnum { if(AnsprechSchluesselVertragEnum.NR_0 == null) AnsprechSchluesselVertragEnum.NR_0 = new AnsprechSchluesselVertragEnum(1, "Bankfreie Betreuung._t F\u00fcr Werbung ansprechbar"); return AnsprechSchluesselVertragEnum.NR_0; };
```
The Umlaut 'ü' is encoded as ANSI in the comment, but it is escaped as '\u00fc' in the literal string.
Since we have some developers working in Linux and some on Windows, this results in diffs and mixed encoding when merging. 
We managed to work around it by putting `-Dfile.encoding=UTF8` into our `.mvn/jvm.config`, but I think this should be fixed at some point.

**Comment by renaudpawlak** [comment]
Yep. Thanks for reporting this issue. It should be fixed when time allows it.

**Comment by lgrignon** [comment]
I might give a look if I find time. What do you think we should do? Use \u unicode notation in comments as well?

**Comment by ADegele** [comment]
\u does not work in comments, at least not in VS Code. I propose specifying the encoding when opening a output stream. For example: `OutputStreamWriter(OutputStream out, Charset cs)`
